src/server/game/Handlers/ChatHandler.cpp

```diff
@@ -218,20 +218,17 @@ void WorldSession::HandleMessagechatOpcode(WorldPacket& recvData)
                         // allow two side chat at group channel if two side group allowed
                         if (sWorld->getBoolConfig(CONFIG_ALLOW_TWO_SIDE_INTERACTION_GROUP))
                             lang = LANG_UNIVERSAL;
-
-                        specialMessageLimit = 35;
                         break;
                     case CHAT_MSG_GUILD:
                     case CHAT_MSG_OFFICER:
                         // allow two side chat at guild channel if two side guild allowed
                         if (sWorld->getBoolConfig(CONFIG_ALLOW_TWO_SIDE_INTERACTION_GUILD))
                             lang = LANG_UNIVERSAL;
-
-                        specialMessageLimit = 15;
                         break;
+
                     case CHAT_MSG_WHISPER:
                         if (sender->getLevel() >= 80)
-                            specialMessageLimit = 15;
+                            lang = LANG_UNIVERSAL;
```

Comment:
why?

Reply:
before we set the rate to 15 if the player is over level 80, this is a azerothcore only thing, as i dont see it in other cores. so when we remove the specialMessageLimit we end up being left with essentially a if statement that does nothing, so we me make it consistent with the line further down so it can be throttled like the rest..
```
-            sender->UpdateSpeakTime(specialMessageLimit);
+            sender->UpdateSpeakTime(lang == LANG_ADDON ? Player::ChatFloodThrottle::ADDON : Player::ChatFloodThrottle::REGULAR);
```

Reply:
yeah but why lang universal? Would it affect to two side interaction? If we set lang to universal, would it mean horde and alliance can talk freely on whispers while the two interaction config is set to 0?

Reply:
🤷if you are above level 80 it would already be beyond the max level for wraith.  Tbh i think we should remove the chat whisper case all together if on perfers. I really dont see the point in adding a above 80 lvl case.

Reply:
@Nefertumm looking over it and staring at the screen for 5 mins. i removed the case for it. Zero sense for its exsistence, no other core has a if statement like that for two sided interactions.